cmd/monitoring/main.go

```diff
@@ -41,7 +42,16 @@ func main() {
 		log.Fatalf("failed to prepare config_set schema with error: %v", err)
 	}
 
-	producer, err := monitoring.NewProducer(ctx, cfg.Kafka)
+	producer, err := monitoring.NewProducer(ctx, cfg.Kafka, cfg.Kafka.Topic)
+	if err != nil {
+		log.Fatalf("failed to create kafka producer with error: %v", err)
+	}
+
+	telSchema, err := schemaRegistry.EnsureSchema(fmt.Sprintf("%s-value", cfg.Kafka.TelemetryTopic), monitoring.TelemetryAvroSchema)
+	if err != nil {
+		log.Fatalf("failed to prepare telemetry_config_set schema with error: %v", err)
+	}
+	telemetryProducer, err := monitoring.NewProducer(ctx, cfg.Kafka, cfg.Kafka.TelemetryTopic)
```

Comment:
I find this namic confusing: both `tel` and `telemetry`. They are also not telemetry, telemetry is done in offchain reporting in the `chainlink` repo. How about using `configSetSchema` and `configSetSimplifiedSchema`?